tests/openthread_ncp/README.md

```diff
@@ -0,0 +1,45 @@
+## OpenThread on RIOT
+
+This test demonstrates how to use OpenThread stack from [OpenThread](https://github.com/openthread/openthread) on
```

Comment:
This is copy pasted from openthread test. Might not be very useful, you can link to the other README file directly (See [openthread test](../openthread/README.md) for more detail).